### ISSUE TITLE
Translations of the new site

### ISSUE THREAD

**Issue by lblissett** [lblissett/dn-discuss#9]
**First of all: we need to have a clear vision of the wording in english before we start to translate!**

What needs to be translated: 

#### German
- [ ] Menus: 
  - [ ] Main Navigation
  - [ ] Footer
- [ ] Teasers: 
  - [ ] Frontpage
  - [ ] Eventpage
  - [ ] Newspage
  - [ ] Grouppage
- [ ] Filters for sorting
  - [ ] Eventpage
  - [ ] Newspage
  - [ ] Publicationpage
- [ ]  Fieldlabels in forms to create new content
  - [ ] Events
  - [ ] Article
  - [ ] Groups
  - [ ] References

#### French
- [ ] Menus: 
  - [ ] Main Navigation
  - [ ] Footer
- [ ] Teasers: 
  - [ ] Frontpage
  - [ ] Eventpage
  - [ ] Newspage
  - [ ] Grouppage
- [ ] Filters for sorting
  - [ ] Eventpage
  - [ ] Newspage
  - [ ] Publicationpage
- [ ]  Fieldlabels in forms to create new content
  - [ ] Events
  - [ ] Article
  - [ ] Groups
  - [ ] References

**Comment by jangermuller** [comment]
Jan, Jaspal, once the English pages have been properly cleaned, you can ask the translators?

**Comment by SinghJN** [comment]
Yes, noted. Once we have cleaned up the English version, I can send out emails to the language administrators.

**Comment by jangermuller** [comment]
Looks great. The only question I have now is do we need to be careful with the URL aliases? What I have been doing now is /GROUPNAME/PAGENAME for example but what path will we have for translated files? Should we discuss a policy for the creation of URL aliases?
